From 803160bb16b8a5409e9eedc7017d98e163ecf362 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Sat, 31 Dec 2011 13:43:00 +0800
Subject: [PATCH 0828/2463] ENGR00171111 MX6Q/TOUCHSCREEN : change the printk
 to dynamic print

Too much printk make the dmesg mess.
Change it to dynamic print mode.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 drivers/input/touchscreen/egalax_ts.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/input/touchscreen/egalax_ts.c b/drivers/input/touchscreen/egalax_ts.c
index 58273b9..fff8afe 100644
--- a/drivers/input/touchscreen/egalax_ts.c
+++ b/drivers/input/touchscreen/egalax_ts.c
@@ -91,7 +91,7 @@ retry:
 
 	dev_dbg(&client->dev, "recv ret:%d", ret);
 	for (i = 0; i < MAX_I2C_DATA_LEN; i++)
-		printk(KERN_DEBUG " %x ", buf[i]);
+		dev_dbg(&client->dev, " %x ", buf[i]);
 
 	if (buf[0] != REPORT_MODE_VENDOR
 	    && buf[0] != REPORT_MODE_SINGLE
-- 
1.7.7.4

